packages/miniflare/package.json

```diff
@@ -47,6 +47,7 @@
 		"acorn-walk": "8.3.2",
 		"exit-hook": "2.2.1",
 		"glob-to-regexp": "0.4.1",
+		"postal-mime": "^2.4.3",
```

Comment:
This should be a `devDependency` instead (so that it's bundled into the Miniflare release)